metron-platform/metron-solr/src/test/java/org/apache/metron/solr/integration/SolrUpdateIntegrationTest.java

```diff
@@ -186,4 +195,114 @@ public void testHugeErrorFields() throws Exception {
     exception.expectMessage("Document contains at least one immense term in field=\"error_hash\"");
     getDao().update(errorDoc, Optional.of("error"));
   }
+
+  @Test
+  @Override
+  public void test() throws Exception {
```

Comment:
What do these changes have to do with ES DAO read/write approach change? This class is testing Solr - should this be in a separate PR?

Reply:
This test was previously shared in UpdateIntegrationTest between ES and Solr.  With these changes, the tests don't behave exactly the same anymore.  That being said, what I did here doesn't look right.   I'll dig into this and see what is going on.

Reply:
Ok, I found the problem. With my changes, the test data was getting loaded into both the ElasticsearchDao and the HBaseDao because I was using a MultiIndexDao to load the test data.  

In master, only Elasticsearch gets loaded with the test data.  This was causing some of the test assumptions to fail.  I corrected how the data is loaded so it only loads Elasticsearch and matches the existing behavior.

Thanks for pointing this out.